src/main/java/appeng/fluids/parts/PartFluidStorageBus.java

```diff
@@ -0,0 +1,508 @@
+
+				this.handler.setBaseAccess( (AccessRestriction) this.getConfigManager().getSetting( Settings.ACCESS ) );
+				this.handler.setWhitelist( this.getInstalledUpgrades( Upgrades.INVERTER ) > 0 ? IncludeExclude.BLACKLIST : IncludeExclude.WHITELIST );
+				this.handler.setPriority( this.getPriority() );
+
+				final IItemList<IAEFluidStack> priorityList = AEApi.instance().storage().getStorageChannel( IFluidStorageChannel.class ).createList();
+
+				final int slotsToUse = 18 + this.getInstalledUpgrades( Upgrades.CAPACITY ) * 9;
+				for( int x = 0; x < this.config.getSlots() && x < slotsToUse; x++ )
+				{
+					final IAEItemStack is = this.config.getAEStackInSlot( x );
+					if( is != null )
+					{
+						// Because we store filtered fluid as buckets, we need to grab the fluid from the stack
+						IFluidHandlerItem fh = FluidUtil.getFluidHandler( is.createItemStack() );
+						if( fh == null )
+						{
+							continue;
+						}
+						FluidStack fluid = fh.drain( Integer.MAX_VALUE, false );
+						priorityList.add( AEFluidStack.fromFluidStack( fluid ) );
+					}
+				}
+
+				if( this.getInstalledUpgrades( Upgrades.FUZZY ) > 0 )
+				{
+					this.handler.setPartitionList( new FuzzyPriorityList<IAEFluidStack>( priorityList, (FuzzyMode) this.getConfigManager().getSetting( Settings.FUZZY_MODE ) ) );
+				}
+				else
+				{
+					this.handler.setPartitionList( new PrecisePriorityList<IAEFluidStack>( priorityList ) );
+				}
+
+				if( inv instanceof IBaseMonitor )
+				{
+					( (IBaseMonitor<IAEFluidStack>) inv ).addListener( this, this.handler );
+				}
+			}
+		}
+
+		// update sleep state...
+		if( wasSleeping != ( this.monitor == null ) )
+		{
+			try
+			{
+				final ITickManager tm = this.getProxy().getTick();
+				if( this.monitor == null )
+				{
+					tm.sleepDevice( this.getProxy().getNode() );
+				}
+				else
+				{
+					tm.wakeDevice( this.getProxy().getNode() );
+				}
+			}
+			catch( final GridAccessException ignore )
+			{
+				// :(
+			}
+		}
+
+		try
+		{
+			// force grid to update handlers...
+			this.getProxy().getGrid().postEvent( new MENetworkCellArrayUpdate() );
+		}
+		catch( final GridAccessException ignore )
+		{
+			// :3
+		}
+
+		return this.handler;
+	}
+
+	private void checkInterfaceVsStorageBus( final TileEntity target, final AEPartLocation side )
+	{
+		IInterfaceHost achievement = null;
+
+		if( target instanceof IInterfaceHost )
+		{
+			achievement = (IInterfaceHost) target;
+		}
+
+		if( target instanceof IPartHost )
+		{
+			final Object part = ( (IPartHost) target ).getPart( side );
+			if( part instanceof IInterfaceHost )
+			{
+				achievement = (IInterfaceHost) part;
+			}
+		}
+
+		if( achievement != null && achievement.getActionableNode() != null )
+		{
+			// Platform.addStat( achievement.getActionableNode().getPlayerID(), Achievements.Recursive.getAchievement()
+			// );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)